### PR TITLE
config/dtb: samsung-heatqlte: Add panel s6288a0 drivers

### PR DESCRIPTION
**Pull request by garethppls**
Add panel drivers for Samsung Galaxy Ace 4 (SM-G357FZ)

### CODE REVIEW

File: config/samsung-heatqlte.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# DTB: From AP_G357FZXXS1AQA2_CL572060_QB12156321_REV00_user_low_ship_MULTI_CERT.tar
+
+OPTIONS=(-r vddio -r vdd --backlight-fallback-dcs --dcs-no-get-brightness)
```
Comment (stephan-gh) [inline config/samsung-heatqlte.sh:4]: The DCS backlight control (sadly) probably doesn't work on Samsung AMOLED panels, so this should probably be `-r vddio -r vdd --no-backlight`.

Comment (garethppls) [reply config/samsung-heatqlte.sh]: Thanks. I've updated this.

**Review by stephan-gh — APPROVED** [review]
Looks fine (preliminary). @wonderfulShrineMaidenOfParadise can you update the Linux part with this?

**Comment by garethppls** [comment]
I guess @wonderfulShrineMaidenOfParadise will have to put this in the history before the commit where we change the colours.

**Comment by wonderfulShrineMaidenOfParadise** [comment]
Done.

**Comment by stephan-gh** [comment]
Pushed, thanks!

**Pull request closed without merging**